modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/PlaybackManager.kt

```diff
@@ -365,10 +365,28 @@ open class PlaybackManager @Inject constructor(
         return player?.supportsVolumeBoost() ?: false
     }
 
+    suspend fun onSwitchedToMeteredConnection() {
+
```

Comment:
Extra line space:)

Reply:
This was actually intentional. I prefer how the extra line space makes the early return easier to see. If there are strong opinions against this though, I can certainly remove it.

Reply:
Turns out that with our updates to Spotless it also doesn't like the extra space, so it's removed now. 😄